persistence-dse-6.8/pom.xml

```diff
@@ -10,7 +10,7 @@
   <groupId>io.stargate.db.dse</groupId>
   <artifactId>persistence-dse-6.8</artifactId>
   <properties>
-    <dse.version>6.8.2-69139b3</dse.version>
+    <dse.version>6.8.5</dse.version>
```

Comment:
It would be nice to bump DSE version in integration tests (here on lin 185 and in `testing/pom.xml`)

Reply:
Do we also need to update ccm?

Reply:
good point. `ccm` itself should be fine, but we need to pre-download the right DSE version

Reply:
Right, that's what I meant. The pre-loaded DSE version.

Reply:
@dougwettlaufer : will you re-build the docker image when this is merged?

Reply:
Sure thing 👍